samples/stock/demo/step-line/demo.js

```diff
@@ -1,28 +1,71 @@
 (async () => {
+    Highcharts.Templating.helpers.lastValue = function () {
+        const points = arguments[0].ctx.points;
+        return points[points.length - 1].y.toFixed(2);
```

Comment:
This causes a crash when toggling between desktop and mobile views in the utils. 

`Cannot read properties of undefined (reading 'y')`